style.css

```diff
@@ -655,3 +694,343 @@ label {
   width: 90px;
   text-align: right;
 }
+
+.link-button:hover,
+.link-button2:hover,
+.resume:hover,
+.sub-bot:hover {
+  color: blue;
+  background: wheat;
+  border: blue;
+}
+
+@media screen and (min-width: 768px) {
+  main {
+    width: 768px;
+    align-items: center;
+  }
+
+  body {
+    display: flex;
+    align-items: center;
+    flex-direction: column;
+  }
+
+  .page1 {
+    background-image: url("/img/desktop-ver.png");
+    background-repeat: no-repeat;
+    width: 1440px;
+    height: 852px;
+    display: flex;
+    flex-direction: row;
+    justify-content: left;
+  }
+
+  .envelope {
+    margin-top: -20px;
+  }
+
+  .social-media {
+    order: 1;
+    align-self: center;
+    grid-column: 2/3;
+    grid-row: 3/4;
+    display: flex;
+    flex-direction: row;
+  }
+
+  .main-header {
+    width: 1440px;
+    height: 72px;
+  }
+
+  .desktop-main {
+    order: 2;
+  }
+
+  .desktop-header {
+    height: 20px;
+    font-family: 'Inter', sans-serif;
+    font-style: normal;
+    font-weight: 600;
+    font-size: 15px;
+    line-height: 20px;
+    color: #fff;
+    column-gap: 30px;
+    justify-content: space-around;
+    margin-right: 70px;
+    display: flex;
+    text-align: center;
+    align-self: center;
+  }
+
+  .union {
+    display: none;
+  }
+
+  .icon {
+    display: flex;
+    flex-direction: column;
+    row-gap: 15px;
+  }
+
+  .recent-work {
+    width: 1440px;
+    align-items: center;
+    display: flex;
+    flex-direction: column;
+  }
```

Comment:
- [ ] Good job designing your projects @mohisa302, Although since you used a fixed size your section becomes a bit different from a Figma design on a different screen size.
- [ ] Also, As You can see your design can use more styled to align with the Figma design. This includes the display of different screen sizes, colors, and background decorations. Please make sure to add more style to better design your `Featured project` section.
- [ ] Dear @mohisa302. Although you add an image I can see you used a '/' to refer to the source. This will only work for a local repository. Please instead use `./` to refer to a relative path. 
> EG.  `src="/img/env.png"` to `src="./img/env.png"`

### Figma design
![Screenshot 2022-12-08 192407](https://user-images.githubusercontent.com/69077061/206506172-50fb6476-115d-480a-a526-85e9e3c5721f.png)

### Your design
![Screenshot 2022-12-08 192347](https://user-images.githubusercontent.com/69077061/206506185-b8633ab9-96f1-4427-97b1-624319e8a8da.png)